### PR TITLE
Misc queue from 2018 (sorry, lost sources) 

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -12596,14 +12599,18 @@ rythim->rhythm
 rythm->rhythm
 rythmic->rhythmic
 rythyms->rhythms
+sabatour->saboteur
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Add sabatage->sabotage

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -12614,6 +12621,9 @@ sais->says
 salery->salary
 samll->small
 samller->smaller
+sammon->salmon
+samori->samurai
+samue->same
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Add Samuel as a suggestion, although this feels like deja vu from another PR!

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done.

**Pull request merged**